packages/skatejs/src/with-update.js

```diff
@@ -133,11 +135,13 @@ export const withUpdate = (Base: Class<any> = HTMLElement): Class<any> =>
     static _attributeToPropertyMap: Object;
     static _observedAttributes: Array<string>;
     static _props: Object;
+    static _propsList: Array<string>;
```

Comment:
I made `_propsList` `static` instead of an instance variable so there's only one instance.